extension/chrome/settings/index.ts

```diff
@@ -50,6 +50,9 @@ View.run(class SettingsView extends View {
     const uncheckedUrlParams = Url.parse(['acctEmail', 'page', 'pageUrlParams', 'advanced', 'addNewAcct']);
     this.acctEmail = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'acctEmail');
     this.page = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'page');
+    if (!/^(\/chrome|modules)/.test!(this.page as string)) {
+      this.page = undefined;
```

Comment:
Please show a modal in this case, to say that an unexpected value was found in the "page" parameter. We could even write a test for this.

Reply:
Modal added! It looks like this:

![Screenshot from 2021-02-03 01-50-56](https://user-images.githubusercontent.com/44826516/106721932-930b8a80-65c2-11eb-80c8-e37ac8360153.png)

Adding a test now, in the meantime let me know if you'd like any changes to the modal